microsoft/testsuites/performance/storageperf.py

```diff
@@ -187,6 +189,104 @@ def perf_storage_over_nfs_synthetic_tcp_4k(self, result: TestResult) -> None:
     def perf_storage_over_nfs_synthetic_udp_4k(self, result: TestResult) -> None:
         self._perf_nfs(result, protocol="udp")
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test data disk performance.
```

Comment:
it is not data disk, if run against azure vm, it is os disk, and it actually depends on where the testing run.

Reply:
Yes this will run on OS disk of L2 guest o2 L2 host.